package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "axios-har-tracker",
-  "version": "1.0.0",
+  "version": "0.1.0-alpha",
   "description": "",
   "main": "dist/axios-har-tracker.js",
   "scripts": {
-    "prebuild": "rimraf ./dist",
-    "build": "tsc"
+    "prebuild": "rimraf ./dist ./coverage ./testresults ./*.har",
+    "build": "tsc",
+    "test": "jest",
+    "posttest": "node ./scripts/jest-join-coverage.js"
```

Comment:
Add `prepublish` script that will execute `npm run build` to avoid publishing package without building dist from latests src